### ISSUE TITLE
Legacy4J crashes on bootup as of 1.4.3

### ISSUE THREAD

**Issue by BuddarScotchy** [Wilyicaro/Legacy-Minecraft#121]
Attempted to boot the game with Legacy4J installed, immediately resulted in a crash
Included is the error message:

![image](https://github.com/Wilyicaro/Legacy-Minecraft/assets/158768860/8de4a51e-5441-4034-bfdf-3fd0cfdd74e5)

And the stacktrace:

![image](https://github.com/Wilyicaro/Legacy-Minecraft/assets/158768860/02a8e304-863c-4de8-8e84-9937217d5346)

**Comment by BuddarScotchy** [comment]
[crash-2024-02-28_03.00.57-client.txt](https://github.com/Wilyicaro/Legacy-Minecraft/files/14430620/crash-2024-02-28_03.00.57-client.txt)
 Here's the full crash log

**Comment by UMFD** [comment]
same here, seems like the simple controller support added in 1.4.3 is causing this, im getting many errors related to SDL controller API
```
Caused by: java.lang.NoClassDefFoundError: com/badlogic/gdx/jnigen/JniGenSharedLibraryLoader
	at org.libsdl.SDL.<clinit>(SDL.java:13)
	at com.studiohartman.jamepad.ControllerManager.nativeInitSDLGamepad(ControllerManager.java:99)
	at com.studiohartman.jamepad.ControllerManager.initSDLGamepad(ControllerManager.java:75)
	at wily.legacy.client.controller.ControllerHandler.<init>(ControllerHandler.java:30)
	at wily.legacy.LegacyMinecraftClient.init(LegacyMinecraftClient.java:125)
	at wily.legacy.fabric.LegacyMinecraftFabricClient.onInitializeClient(LegacyMinecraftFabricClient.java:10)
	at net.fabricmc.loader.impl.FabricLoaderImpl.invokeEntrypoints(FabricLoaderImpl.java:384)
	... 9 more
```

**Comment by Wilyicaro** [comment]
Fixed in 1.4.3a version
